index.html

```diff
@@ -1,69 +1,269 @@
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+              </ul>
+              <div>
+                <a class="button_primary" href="#">See project</a>
+              </div>
+            </div>
+          </article>
+          <article class="work_card">
+            <header class="work__header">
+              <img
+                src="assets/snapshoot4.png"
+                alt="Snapshoot Portfolio 4"
+                class="work__image"
+              />
+            </header>
+            <div class="portfolio__content">
+              <h3 class="portfolio__title color-dark">Multi-Post Stories</h3>
+              <div class="portfolio__profile">
+                <h4 class="portfolio__author color-info">Canopy</h4>
+                <ul class="portfolio__list">
+                  <li class="work__item">Back End Dev</li>
+                  <li class="work__item">2015</li>
+                </ul>
+              </div>
+              <p class="portfolio__description">
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+              <ul class="portfolio__tags">
+                <li class="tag">HTML</li>
+                <li class="tag">CSS</li>
+                <li class="tag">JavaScript</li>
+              </ul>
+              <div>
+                <a class="button_primary" href="#">See project</a>
+              </div>
+            </div>
+          </article>
+        </div>
+      </section>
+      <section class="section section_white" id="about_me">
+          <div class="about">
+            <h2 class="about_title">About Myself</h2>
+            <p class="color-info">
+              Hello I'm a software developer! I can help you build a product,
+              feature or website. Look through some of my work and experience!
+              If you like what you see and have a project you need coded,
+              don't hesitate to contact me.
+            </p>
+            <h2 class="connect">LET’S CONNECT</h2>
+            <ul class="social_icons">
+              <li><a href=""><img src="assets/twitter.png" alt="Twitter" /></a></li>
+              <li><a href=""><img src="assets/Linkedin.png" alt="LinkedIn" /></a></li>
+              <li><a href=""><img src="assets/medium.png" alt="Medium" /></a></li>
+              <li><a href=""><img src="assets/github.png" alt="Github" /></a></li>
+              <li><a href=""><img src="assets/angel.png" alt="Angel" /></a></li>
+            
+            </ul>
+            </div>
+            <div>
+              <a  class="button_primary">Get my resume</a>
+            </div>
+            <div class="category-about_page">
+              <header
+                class="about_page_header flex-row justify-between align-center"
+              >
+                <h3 class="about_page_subtitle">Languages</h3>
```

Comment:
- [Optional] Kindly consider adjusting the alignment of some of the features of this section which I've highlighted in the image below to have a replica of what's provided on the Figma design. You'll agree with me it does not give a better user experience 👍 

### Your work:
![image](https://github.com/montsecruz9/personal-portfolio/assets/50990925/d167ec16-0bec-4f9b-b1f4-d5c0e06bffd9)

### Figma:
![image](https://github.com/montsecruz9/personal-portfolio/assets/50990925/c99200ef-16f7-45a1-9a4a-cb6a69bd134b)